client/sites-dashboard/components/sites-grid-item.tsx

```diff
@@ -198,13 +202,26 @@ export const SitesGridItem = memo( ( props: SitesGridItemProps ) => {
 						<Truncated>{ displaySiteUrl( siteUrl ) }</Truncated>
 					</SiteUrl>
 					<WithAtomicTransfer site={ site }>
-						{ ( result ) =>
-							result.wasTransferring ? (
-								<TransferNoticeWrapper { ...result } />
-							) : (
-								<>{ showLaunchNag && <SiteLaunchNag site={ site } /> }</>
-							)
-						}
+						{ ( result ) => {
+							if ( result.wasTransferring ) {
+								return <TransferNoticeWrapper { ...result } />;
+							} else if ( showLaunchNag && 'unlaunched' === site.launch_status ) {
```

Comment:
I initially wondered if this check was redundant, but looking further it seems `showLaunchNag` is generally filled with a basic boolean that doesnt check if the site is launched, and the component returns `null` if the site is launched.

Long story short, this unlaunched check makes sense.

Reply:
yea it took me a while to figure this out 😆